android/app/src/main/java/me/rainbow/NativeModules/Haptics/RNHapticsPerformer.java

```diff
@@ -0,0 +1,50 @@
+
+package me.rainbow.NativeModules.Haptics;
+
+import android.os.Build;
+import android.os.VibrationEffect;
+import android.os.Vibrator;
+
+class RNHapticsPerformer {
+  private final Vibrator mVibrator;
+  RNHapticsPerformer(Vibrator vibrator) {
+    mVibrator = vibrator;
+  }
+
+  void notificationSuccess() {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+      mVibrator.vibrate(VibrationEffect.createWaveform(new long[]{20, 65, 21}, new int[]{145, 0, 130}, -1));
```

Comment:
Where are those numbers coming from? Do we care about potentially editing them in future?

Reply:
my intuition